next.config.js

```diff
@@ -1,14 +1,10 @@
-const withCSS = require("@zeit/next-css");
+const withPlugins = require("next-compose-plugins");
 const withImages = require("next-images");
```

Comment:
Why not use next/image? Just curious 🙏

Reply:
You shouldn’t need the same optimization for SVGs as other images because they’re vector based. Does next-images optimize the SVGs or something?

Reply:
I don't think that it does.
I have tried to use next/image and it appears that this code does not work:
```
import Loader from "@/images/loader.svg";

<Image src={Loader} />
```
I get this error
```
error - ./images/loader.svg 1:0
Module parse failed: Unexpected token (1:0)```

Reply:
Yeah you shouldn’t even need to use it with SVGs. Could you try that instead? Next-images is a community maintained package and not controlled by the Next.js team.

Reply:
I don't quite understand what you mean. I am open to using any other solution but when I try to use next/image it does not work. Check the code snippet in my previous answer

Reply:
Whoops, I totally looked at that wrong! So the issue is you’re directly importing the SVG file. Instead, place the svg file in your public folder and then use the path as a string for the src.

Reply:
Thanks! I have made this change and now it works without the plugin. I have pushed the changes

Reply:
Amazing. Thank you so much!